### PR TITLE
Font Change

### LINKED ISSUE #1: [Bug]: Fix the Design Difference
**Change Download Window**
The Design of  Downloads Window is different from the original site.

**Expected behavior**
image

**Current Behaviour**
image

### PR DESCRIPTION
**Pull request by Sehaj-kahlon**
## Description
Added different font 

## Issue Resolved
 Fixes #1.

## Screenshots 

image

### CODE REVIEW

**Review by Manobal-Singh-Bagady — APPROVED** [review]
The Changes Looks Good to me. Good Work, @Sehaj-kahlon 👍, Please make a comment Comment the issue thread #1 so that I can assign the issue to you.

**Review by Manobal-Singh-Bagady — CHANGES_REQUESTED** [review]
Please update the version to the latest version too. Sorry that was not mentioned in the issue.

File: index.html [hunk 1]
```diff
@@ -135,7 +135,7 @@ <h3>Community</h3>
           <!-- Monitor for downloads -->
           <div class="monitor windows">
             <h4>Latest Resource Release</h4>
-            <span class="version"> 2.39.0 </span>
+            <span class="version">2.39.0</span>
```
Comment (Manobal-Singh-Bagady) [inline index.html]: Please update the version from 2.39.0 to the current latest version.

**Comment by Sehaj-kahlon** [comment]
do i have to update the release date too??

**Comment by Manobal-Singh-Bagady** [comment]
> do i have to update the release date too??

Yeah sure you can do that too but please first comment on #1 so that i can assign you this issue.

**Comment by Manobal-Singh-Bagady** [comment]
Alright I have assigned the issue to you, please complete the requested changes and i will merge the pull request 👍

**Comment by Sehaj-kahlon** [comment]
image

## updated the version and release date

**Review by Manobal-Singh-Bagady — APPROVED** [review]
Ok, Everything Looks Good to me.  Good Work, @Sehaj-kahlon. I am merging the Pull Request.

**Pull request merged**